consoleme/handlers/base.py

```diff
@@ -444,7 +444,7 @@ async def authorization_flow(
             log_data[
                 "message"
             ] = "No eligible roles detected for user. But letting them continue"
-            log.error(log_data)
+            log.warn(log_data)
```

Comment:
Nit: `log.warn()` is deprecated in favor of `log.warning()`

Reply:
thanks, fixed. added pre-commit to detect future usages of log.warn and fixed other usages in the repo